diff --git a/source/libraries/ContractCompiler.ts b/source/libraries/ContractCompiler.ts
index a56a530..18b39de 100644
--- a/source/libraries/ContractCompiler.ts
+++ b/source/libraries/ContractCompiler.ts
@@ -82,7 +82,7 @@ export class ContractCompiler {
             }

             if (errors.length > 0) {
-                throw new Error("The following errors/warnings were returned by solc:\n\n" + errors);
+                console.log("The following errors/warnings were returned by solc:\n\n" + errors);
             }
         }

